crawl-ref/source/dat/des/arrival/large.des

```diff
@@ -53,7 +53,7 @@ xxxxx''........xx...''.........''...'..xx...xx
 xxx'....ccccc...xxx.....ccccc......'......x..@
 x....cccc...cccc.....cccc...cccc.......''..xxx
 x..ccc.........ccc+ccc.........ccc....''..xxxx
-x'.c......{......+.+......T......+.........xxx
+x'.c...>..{......+.+......T......+.........xxx
```

Comment:
This is probably better fixed by adding KPROP: no_monster_gen to the ∞ part of the vault (the two stone rooms here).